charts/eks/templates/_helpers.tpl

```diff
@@ -37,7 +41,22 @@ Whether the ingressclasses syncer should be enabled
 Whether to create a cluster role or not
 */}}
 {{- define "vcluster.createClusterRole" -}}
-{{- if or (not (empty (include "vcluster.serviceMapping.fromHost" . ))) (not (empty (include "vcluster.plugin.clusterRoleExtraRules" . ))) .Values.rbac.clusterRole.create .Values.sync.hoststorageclasses.enabled (index ((index .Values.sync "legacy-storageclasses") | default (dict "enabled" false)) "enabled") (include "vcluster.syncIngressclassesEnabled" . ) .Values.sync.ingresses.enabled .Values.sync.nodes.enabled .Values.sync.persistentvolumes.enabled .Values.sync.storageclasses.enabled .Values.sync.priorityclasses.enabled .Values.sync.volumesnapshots.enabled -}}
```

Comment:
@FabianKramm apart from the `vcluster.syncIngressclassesEnabled` I think the main issue was on this line. We had both the entries here, I have removed the `.Values.sync.ingresses.enabled` because its already being checked for (inside `vcluster.syncIngressclassesEnabled`) in conjunction with `.Values.sync.ingressclasses.enabled`
```diff
...
  (include "vcluster.syncIngressclassesEnabled" . )
- .Values.sync.ingresses.enabled
...
```